portal/components/react-admin/customHasura/customVariables.js

```diff
@@ -231,7 +231,7 @@ const buildUpdateVariables =
   };
 
 const buildCreateVariables =
-  (introspectionResults) => (resource, aorFetchType, params, queryType) => {
+  (introspectionResults) => (resource, aorFetchType, params) => {
```

Comment:
Why we removed queryType, was it of no use?

Reply:
In this file they declared but not used.